app/main.py

```diff
@@ -0,0 +1,42 @@
+import torch
+import gradio as gr
+import fortepyan as ff
+from omegaconf import OmegaConf
+from fortepyan.audio import render as render_audio
+
+from app.tools import load_model, predict_velocity
+
+
+def run_app(target: str, midi_file):
+    checkpoint = torch.load("checkpoints/velocity/midi-transformer-2023-10-03-08-34.pt", map_location="cpu")
```

Comment:
1. Can you make it possible to select one of the checkpoints present in the folder?
2. If you have a well performing model in HF, can you add code that will download it?